plugins/plugin-hosted/codenvy-machine-hosted/src/main/java/com/codenvy/machine/agent/WorkspaceInfrastructureModule.java

```diff
@@ -0,0 +1,37 @@
+/*
+ *  [2012] - [2016] Codenvy, S.A.
+ *  All Rights Reserved.
+ *
+ * NOTICE:  All information contained herein is, and remains
+ * the property of Codenvy S.A. and its suppliers,
+ * if any.  The intellectual and technical concepts contained
+ * herein are proprietary to Codenvy S.A.
+ * and its suppliers and may be covered by U.S. and Foreign Patents,
+ * patents in process, and are protected by trade secret or copyright law.
+ * Dissemination of this information or reproduction of this material
+ * is strictly forbidden unless prior written permission is obtained
+ * from Codenvy S.A..
+ */
+package com.codenvy.machine.agent;
+
+import com.google.inject.AbstractModule;
+import com.google.inject.multibindings.MapBinder;
+
+import org.eclipse.che.api.environment.server.InfrastructureProvisioner;
+
+/**
+ * @author Alexander Garagatyi
+ */
+public class WorkspaceInfrastructureModule extends AbstractModule {
```

Comment:
I don't think this name is good, first time i saw it i thought it would bind all the workspace related components